examples/vl3_basic/helm/vl3/values.yaml

```diff
@@ -10,7 +10,7 @@ pullPolicy: Always
 cnns:
   nsr:
     port: 5005
-    # addr: 
+    addr: localhost
```

Comment:
don't change this...  `localhost:5005` means something different than `:5005`

Reply:
yes I know this, I did this change because we have this condition `{{- if .Values.cnns.nsr.addr }}` in  `vl3-nse-ucnf.tpl` so I used some random value to trigger the rendering of the new annotations